dev/com.ibm.ws.opentracing.cdi/src/com/ibm/ws/opentracing/cdi/TracedInterceptor.java

```diff
@@ -0,0 +1,34 @@
+/*******************************************************************************
+ * Copyright (c) 2017 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package com.ibm.ws.opentracing.cdi;
+
+import javax.annotation.Priority;
+import javax.interceptor.AroundInvoke;
+import javax.interceptor.Interceptor;
+import javax.interceptor.InvocationContext;
+
+import org.eclipse.microprofile.opentracing.Traced;
+
+@Traced
+@Interceptor
+@Priority(Interceptor.Priority.LIBRARY_BEFORE) //run this interceptor after platform interceptors but before application interceptors
+public class TracedInterceptor {
+
+    @AroundInvoke
+    public Object executeFT(InvocationContext context) throws Throwable {
```

Comment:
you might want to Rename this method to ececuteTraced instead.

Reply:
do you want tracer to be before FT interceptor? If yes, you should do use AfterPlatform + 1 to ensure it is happen first.